python/sglang/srt/mem_cache/storage/mooncake_store/mooncake_store.py

```diff
@@ -1,16 +1,19 @@
 import json
 import logging
 import os
+import time
 import uuid
 from dataclasses import dataclass
 from typing import Any, List, Optional
+import requests
 
 import torch
 
 from sglang.srt.mem_cache.hicache_storage import HiCacheStorage, HiCacheStorageConfig
 
 DEFAULT_GLOBAL_SEGMENT_SIZE = 4 * 1024 * 1024 * 1024  # 4 GiB
 DEFAULT_LOCAL_BUFFER_SIZE = 16 * 1024 * 1024  # 16 MB
+SETUP_TIMEOUT = 600 # 10min
```

Comment:
Ten minutes is too long to start Mooncake Master. I guess 60 seconds might be preferable.